report/assets/styles.css

```diff
@@ -1496,14 +1495,20 @@
 }
 
 .lh-table {
-  border-collapse: collapse;
+  position: relative;
+  border-collapse: separate;
+  border-spacing: 0;
   /* Can't assign padding to table, so shorten the width instead. */
   width: calc(100% - var(--audit-description-padding-left) - var(--stackpack-padding-horizontal));
   border: 1px solid var(--report-border-color-secondary);
-
 }
 
 .lh-table thead th {
+  position: sticky;
```

Comment:
i would have expected to see these on the `thead` instead.  i don't know if it makes a diff to the rendering engine but it feels nicer to manage the position of these as a group rather than individually. 
wdyt?

Reply:
positioning that element does not work, it's a special display element

Reply:
Sorry, got bit confused here. thead support `position: sticky` but... border-bottom seemed to not work.

Reply:
True. I had thought I got it working, but couldn't reproduce it without losing the border.
All good!